bqemulatormanager/emulator.py

```diff
@@ -17,8 +17,7 @@ def __init__(self, project_name: str, port: int, launch_emulator: bool = True, d
             if is_port_in_use(port):
                 raise PortOccupiedError(f'port {port} is occupied.')
             self.running_flg = True
-            self.proc = subprocess.Popen(["bigquery-emulator", "--project", f"{project_name}", "--port", f"{port}"
-                                          "--log-level", f"{log_level}"], shell=True)
+            self.proc = subprocess.Popen(["bigquery-emulator", "--project", f"{project_name}", "--port", f"{port}", "--log-level", f"{log_level}"], shell=True)
```

Comment:
re-format only, right ?

Reply:
No, there was not ',' between `f"{port}"` and `"--log-level"`.